### PR TITLE
Add csharp mongodb template to community file

### PR DESCRIPTION
**Pull request by Marcus-Smallman**
Signed-off-by: marcus-smallman <marcussmallman@live.co.uk>

## Description
Updated the community file with the csharp mongodb template.

## Checklist:
- [ ] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: community.md [hunk 1]
```diff
@@ -140,6 +140,7 @@ Key:
 
 | Blog/repo name and description                                          | Author       | Site     | Date        |
 |---------------------------------------------------------------------|--------------|----------|-------------|
+| [.Net Core – OpenFaas – MongoDB](https://marcussmallman.io/2018/11/27/net-core-openfaas-mongodb/) | Marcus Smallman | marcussmallman.io | 28-Nov-2018 |
```
Comment (alexellis) [inline community.md]: The README is very good but the blog is a bit sparse. Can you bring some of they content over like the handler example?

Armhf means Raspberry Pi, so I'm not sure if that's what you meant?

Comment (Marcus-Smallman) [reply community.md]: I have updated the blog post, I'd be happy to add anything that could have been missed too.

Regarding **armhf**, I have a raspberry pi cluster for which I am deploying my functions too, but am developing them using .net core on my x86 machine. If you're referencing the description made on the architecture used in your mongo function then I meant design architecture :). I have updated the blog post to describe that and I have added a note on the github project stating the currently supported architectures for the function template.

**Comment by alexellis** [comment]
For us to share it, it'd be ideal if you had a normal template for x86_64 alongside the armhf one. Let me know your thoughts

**Comment by alexellis** [comment]
Checkout the deployment guide for OpenFaaS, it'll show you how to get into the Play with Docker environment to test free on the cloud.

**Comment by Marcus-Smallman** [comment]
Successfully tested in that environment (Play with Docker). The following was completed:

 - Created a mongodb container which the function would use to reference:
```
docker volume create mongodata
docker run --name mongodb -d -p 27017:27017 -v mongodata:/data/db mongo mongod
```
 - Deployed OpenFaas following the deployment guide (https://docs.openfaas.com/deployment/docker-swarm/)

- Created a stack that referenced the x86_64 csharp mongodb function and the online deployment of OpenFaas:

```
provider:
  name: faas
  gateway: http://ip172-18-0-9-bfvgolk3uhdg00bg0a6g-8080.direct.labs.play-with-docker.com

functions:
  mongo-function:
    lang: csharp-kestrel-mongo
    handler: ./mongo-function
    image: marcussmallman/mongo-function
    environment:
      read_timeout: 20
      write_timeout: 20
      mongo_endpoint: 192.168.0.33:27017
      mongo_database_name: my_mongo_database
      mongo_collection_name: my_mongo_collection
```

The function successfully deployed and data writes to mongo:

image

image

**Comment by alexellis** [comment]
Looks good. Maybe you could even reuse your screenshots in your blog post to get a bit more detail across to the users and bump the interest levels up?

**Comment by alexellis** [comment]
Merged 👍 😄

**Comment by Marcus-Smallman** [comment]
I'll see what I can do when I've got some time, thanks for the feedback!

**Comment by Marcus-Smallman** [comment]
Blog post updated! 😄

**Pull request merged**